Zotero/Scenes/Detail/PDF/Views/PDFAnnotationsViewController.swift

```diff
@@ -143,8 +137,18 @@ final class PDFAnnotationsViewController: UIViewController {
             viewModel.process(action: .setComment(key: annotation.key, comment: comment))
 
         case .reloadHeight:
-            updateCellHeight()
-            focusSelectedCell()
+            if let key = viewModel.state.selectedAnnotationKey {
```

Comment:
AFAICS `.reloadHeight` only triggers when the user edits the comment in the sidebar, so it will always be the selected annotation. Popover comment editing doesn't trigger this, as well as sync updates.

Reply:
That's true, but it's enough to just reconfigure this single cell and the table view will move all cells properly? That definitely improved from since I implemented this :).

Anyway I guess we can skip the `else`, but I don't mind it. Also you could extract reconfiguring that key to separate child function.